data.sql

```diff
@@ -1,34 +1,41 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Agumon', '2020-02-03',0,true, 10.23);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Gabumon', '2018-11-15', 2,true, 8);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Pikachu', '2021-01-07', 1,false, 15.04);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Devimon', '2017-05-12', 5,true, 11);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Charmander', '2020-02-08', 0,false, -11);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Plantmon', '2021-11-15', 2,true, -5.7);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Squirtle', '1993-04-02', 3,false, -12.13);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Angemon', '2005-06-12', 1,true, -45);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Boarmon', '2005-06-07', 7,true, 20.4);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Blossom', '1998-10-13', 3,true, 17);
-
-INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
-('Ditto', '2022-05-14', 4,true, 22);
\ No newline at end of file
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith 34', 34), 
```

Comment:
- [ ] You have a typographical error when populating data into the `owners` table for Sam Smith. It should be `Sam Smith` and not `Sam Smith 34`. This affects the results gotten from some queries. **_For example, the query that asks for all owners and their animals, including those that don't own any animal_**. Kindly correct it 👍